src/startup.c

```diff
@@ -156,6 +152,26 @@ void _start_c(void *boot_args, void *base)
     adt =
         (void *)(((u64)cur_boot_args.devtree) - cur_boot_args.virt_base + cur_boot_args.phys_base);
 
+#ifndef BRINGUP
+    int node = adt_path_offset(adt, "/cpus");
+    if (node >= 0) {
+        ADT_FOREACH_CHILD(adt, node)
+        {
+            const char *state = adt_getprop(adt, node, "state", NULL);
+            if (!state)
+                continue;
+            if (strcmp(state, "running") == 0)
+                if (ADT_GETPROP(adt, node, "cpu-id", &cpu_id) == sizeof(cpu_id))
```

Comment:
Please add a fallback to `reg` here, iPhone 5s iOS 10.3.3 ADT does not have `cpu-id`.

Reply:
`cpu_id` is initialized to 0 and it remains untouched if the the CPU nodes have no `cpu-id` property.